src/sentry/static/sentry/app/views/performance/onboarding.tsx

```diff
@@ -1,28 +1,50 @@
 import React from 'react';
+import styled from '@emotion/styled';
 
 import OnboardingPanel from 'app/components/onboardingPanel';
 import Button from 'app/components/button';
-import {IconLightning} from 'app/icons';
+import space from 'app/styles/space';
 import {t} from 'app/locale';
 
+import emptyState from '../../../images/spot/performance-empty-state.svg';
+
 function Onboarding() {
   return (
-    <OnboardingPanel image={<IconLightning size="200px" />}>
-      <h3>{t('No transactions yet')}</h3>
+    <StyledOnboardingPanel>
+      <h3>{t('Pinpoint problems')}</h3>
       <p>
         {t(
-          'View transactions sorted by slowest duration time, related issues, and number of users having a slow experience in one consolidated view. Trace those 10-second page loads to poor-performing API calls and its children.'
+          "You've got this souped up plan. Now what? Get your software set up. We’ve got endpoints to track down."
         )}
       </p>
-      <Button
-        priority="primary"
-        target="_blank"
-        href="https://docs.sentry.io/performance/distributed-tracing/#setting-up-tracing"
-      >
-        {t('Start Setup')}
-      </Button>
-    </OnboardingPanel>
+      <ButtonList>
+        <Button
+          priority="default"
+          target="_blank"
+          href="https://docs.sentry.io/performance-monitoring/setup/"
+        >
+          {t('Take the tour')}
+        </Button>
+        <Button
+          priority="primary"
+          target="_blank"
+          href="https://docs.sentry.io/performance-monitoring/setup/"
+        >
+          {t('Step 1: Install')}
+        </Button>
+      </ButtonList>
+    </StyledOnboardingPanel>
   );
 }
 
+const StyledOnboardingPanel = styled(OnboardingPanel)`
+  background: url(${emptyState}) no-repeat left center !important;
+`;
+
+const ButtonList = styled('div')`
```

Comment:
You could use the `ButtonBar` component for this.

Reply:
@markstory I'd need to style them anyways (`ButtonBar` button's go the full width). Is there any other value?

Reply:
Using a styled button bar would be your best bet if you need a partial width container. Reusing the button bar would reduce some duplication and reduce the amount of generated CSS, but this works.